net/p910nd/files/p910nd.hotplug

```diff
@@ -48,7 +48,7 @@ SYSUPGRADE_CONF="/etc/sysupgrade.conf"
 # DEV_TYPE_FILTER="usb_device"
 
 # Test the script by running $0 -d.
-if [ -n "$1" -a "$1" == "-d" ]; then
+if [ -n "$1" ] && [ "$1" == "-d" ]; then
```

Comment:
== is a bashism. I recommend running through shellcheck.

Reply:
Yes, thanks. Fixed now - revised a bunch of stuff.